storybook/stories/Markdown.js

```diff
@@ -4,6 +4,7 @@ import { ScrollView, StyleSheet, View } from 'react-native';
 import { storiesOf } from '@storybook/react-native';
 
 import Markdown from '../../app/containers/markdown';
+import Preview from '../../app/containers/markdown/Preview';
```

Comment:
Can we remove those extra props?
IPreview should make sure we only send the right ones from now on.